DigitalOffice/Tests/FileServiceTests/Controllers/FIleControllerTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using LT.DigitalOffice.FileService.Commands.Interfaces;
+using LT.DigitalOffice.FileService.Controllers;
+using LT.DigitalOffice.FileService.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.FileServiceUnitTests.Controllers
+{
+    public class CheckRightsControllerTests
```

Comment:
It's FileControllerTests
Also, there's a mistake in file name FIleControllerTests.cs. It's FileControllerTests.cs

Reply:
Thanks. Fixed